util/test/sub_test

```diff
@@ -504,8 +504,18 @@ def get_exec_log_name(execname, comp_opts_count=None, exec_opts_count=None):
 # Use testEnv to process skipif files, it works for executable and
 # non-executable versions
 def runSkipIf(skipifName):
-    skiptest = subprocess.Popen([utildir+'/test/testEnv', './'+skipifName], stdout=subprocess.PIPE).communicate()[0]
-    return skiptest
+    p = subprocess.Popen([utildir+'/test/testEnv', './'+skipifName], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (stdout, stderr) = p.communicate()
+    status = p.returncode
+    stderr = stderr.strip()
+    if status:
+        if stderr:
+            sys.stdout.write(stderr+'\n')
+        Fatal('%s: exit status %d' % (skipifName, status))
```

Comment:
Yes. Commit message:
    This change also makes sub_test exit immediately upon detecting
    such an error. Any Chapel tests in the current test directory which
    haven't already been processed, will never show up in the test
    results at all.
    
    Sub_test already handles some other system-error conditions the
    same way. IMO, it would be too difficult / not worth the effort
    to make sub_test continue working the current test directory in
    these cases.